library_generation/README.md

```diff
@@ -48,6 +48,20 @@ right version for each library.
 Please refer [here](go/java-client-releasing#versionstxt-manifest) for more info
 of versions.txt.
 
+### Api definition path (`api_definition_path`), optional
+
+The path to where the api definition (proto, service yaml) resides.
+
+The default value is the current working directory when running the script.
+
+Note that you need not only the protos defined the service, but also the transitive
```

Comment:
What does the error look like if the dependencies are missing?

Reply:
This is the error message when I remove `google/api` folder in [this commit
](https://github.com/googleapis/sdk-platform-java/pull/3208/commits/69e23dfa069633788bc47fb620beee48d3e1cf9f)

```
google/api/field_behavior.proto: File not found.
google/api/resource.proto: File not found.
google/cloud/alloydb/v1/resources.proto:19:1: Import "google/api/field_behavior.proto" was not found or had errors.
google/cloud/alloydb/v1/resources.proto:20:1: Import "google/api/resource.proto" was not found or had errors.
```

Reply:
Also updated the doc.